CONTRIBUTING.md

```diff
@@ -50,39 +50,39 @@ control logs or otherwise.)
 
 (a) The contribution was created in whole or in part by me and I have
-    the right to submit it, either on my behalf or on behalf of my
-    employer, under the terms and conditions as described by this file;
-    or
+the right to submit it, either on my behalf or on behalf of my
+employer, under the terms and conditions as described by this file;
+or
 
 (b) The contribution is based upon previous work that, to the best of
-    my knowledge, is covered under an appropriate licence and I have
-    the right or permission from the copyright owner under that licence
-    to submit that work with modifications, whether created in whole or
-    in part by me, under the terms and conditions as described by
-    this file; or
+my knowledge, is covered under an appropriate licence and I have
+the right or permission from the copyright owner under that licence
+to submit that work with modifications, whether created in whole or
+in part by me, under the terms and conditions as described by
+this file; or
 
 (c) The contribution was provided directly to me by some other person
-    who certified (a) or (b) and I have not modified it.
+who certified (a) or (b) and I have not modified it.
 
 (d) I understand and agree that this project and the contribution
-    are public and that a record of the contribution (including my
-    name and email address) is retained for the full term of
-    the copyright and may be redistributed consistent with this project
-    or the licence(s) involved.
+are public and that a record of the contribution (including my
+name and email address) is retained for the full term of
+the copyright and may be redistributed consistent with this project
+or the licence(s) involved.
 
 (e) I, or my employer, grant to NIWA and all recipients of
-    this software a perpetual, worldwide, non-exclusive, no-charge,
-    royalty-free, irrevocable copyright licence to reproduce, modify,
-    prepare derivative works of, publicly display, publicly perform,
-    sub-licence, and distribute this contribution and such modifications
-    and derivative works consistent with this project or the licence(s)
-    involved or other appropriate open source licence(s) specified by
-    the project and approved by the
-    [Open Source Initiative (OSI)](http://www.opensource.org/).
+this software a perpetual, worldwide, non-exclusive, no-charge,
+royalty-free, irrevocable copyright licence to reproduce, modify,
+prepare derivative works of, publicly display, publicly perform,
+sub-licence, and distribute this contribution and such modifications
+and derivative works consistent with this project or the licence(s)
+involved or other appropriate open source licence(s) specified by
+the project and approved by the
+[Open Source Initiative (OSI)](http://www.opensource.org/).
 
 (f) If I become aware of anything that would make any of the above
-    inaccurate, in any way, I will let NIWA know as soon as
-    I become aware.
```

Comment:
These changes to `CONTRIBUTING.md` are probably not required for this update.

Reply:
Sorry confused, should I undo the last commits of adding my name?

Reply:
Not that part. You added your name, but probably your editor formatted the whole file, changing some lines (GitHub UI highlights changes that add content as green, and changes that remove content as red, as the lines above my comment)